components/NavBar.js

```diff
@@ -31,6 +31,11 @@ export default function NavBar (props) {
               About
             </NavLink>
           </li>
+          <li>
+            <NavLink href='/status'>
```

Comment:
This gives a 404 error, let's not include it yet.

Reply:
There are some changes in #6 that remove this (and add some links to a footer).